rocketpool-daemon/api/pdao/status.go

```diff
@@ -107,28 +117,32 @@ func (c *protocolDaoGetStatusContext) GetState(mc *batch.MultiCaller) {
 		c.node.Exists,
 		c.node.IsVotingInitialized,
 	)
+
+	// Snapshot Registry
+	if c.registry != nil {
```

Comment:
I ended up removing this check because it's already done here: https://github.com/rocket-pool/smartnode/blob/8a91b72353d7d647fa02202a1e5908f4a57b634f/rocketpool-daemon/api/pdao/status.go#L84